news/feeds.py

```diff
@@ -15,7 +16,9 @@ def item_heading(self, item):
         return item.heading
 
     def item_description(self, item):
-        return item.subheading
+        h = html2text.HTML2Text()
+        excerpt = h.handle(str(item.body)).split('\n\n')[0]
+        return excerpt + "..."
```

Comment:
It seems like this only uses the first line in the body. Should ellipses only be added if there is more than one line? And would there be a case where the first line is "too long" (not sure what that would be)?